models/inventory/StockTransfer.ts

```diff
@@ -246,6 +270,30 @@ export abstract class StockTransfer extends Transfer {
     await invoice.setAndSync('stockNotTransferred', notTransferred);
   }
 
+  async _updateItemsReturned() {
+    if (!this.returnAgainst) {
+      return null;
+    }
+
+    const returnDocs = await this.fyo.db.getAll(this.schema.name, {
+      filters: {
+        returnAgainst: this.returnAgainst,
+        submitted: true,
+        cancelled: false,
+      },
+    });
+
+    const isItemsReturned = returnDocs.length;
+
+    const referenceDoc = await this.fyo.doc.getDoc(
+      this.schema.name,
+      this.returnAgainst
+    );
+
+    await referenceDoc.setAndSync({ isItemsReturned });
```

Comment:
Here `isItemsReturn` is being passed a `number` where as the value is supposed to be a `boolean` cause it's type is `Check`.

Of course it gets fixed by `Converter`, but you still should be passing a `boolean` value to prevent confusion and errors in the future. Remember your code could potentially be here for an indefinite amount of time.

Reply:
corrected